scipy/special/cephes/igam.c

```diff
@@ -207,22 +211,24 @@ double igamc(double a, double x)
  */
 double igam_fac(double a, double x)
 {
+    double ax, fac, res, num;
+
     if (fabs(a - x) > 0.4 * fabs(a)) {
-	double ax = a * log(x) - x - lgam(a);
```

Comment:
Valid, but the change in this PR still looks fine to me - cleaner to declare everything at top of function.